charts/kube-acp-stack/Chart.yaml

```diff
@@ -15,6 +15,6 @@ dependencies:
   repository: https://charts.bitnami.com/bitnami
   condition: redis-cluster.enabled
 - name: acp
-  version: "2.2.1-6"
```

Comment:
Not sure which version should I use here. We'd like to release 2.2.1 but here it's already defined `2.2.1-6`? Any ideas?